Header header
float64 Speed
float64 Steer
float64 Brake
uint8 Gear
bool Handbrake
bool RightLight
bool LeftLight
bool Light
bool SelfDrive
uint8 SpeedMode
uint8 SteerMode
bool Emergency

uint8 Gear_Forward = 1
uint8 Gear_Neutral = 2
uint8 Gear_Reverse = 3

bool Handbrake_Off = 0
bool Handbrake_On = 1

bool Light_Off = 0
bool Light_On = 1

bool SelfDrive_Off = 0
bool SelfDrive_On = 1

uint8 SpeedMode_Slow = 1
uint8 SpeedMode_Fast = 2

uint8 SteerMode_Opposite = 1
uint8 SteerMode_Normal = 0
uint8 SteerMode_Synchro = 2

bool Emergency_Off = 0
bool Emergency_On = 1



